docs/applications/spark.rst

```diff
@@ -11,107 +11,93 @@ Examples
+    val spark = SparkSession.builder()
+      .appName("Neo4j to GraphAr for Movie Graph")
+      .config("neo4j.url", "bolt://localhost:7687")
+      .config("neo4j.authentication.type", "basic")
+      .config("neo4j.authentication.basic.username", sys.env.get("Neo4j_USR").get)
+      .config("neo4j.authentication.basic.password", sys.env.get("Neo4j_PWD").get)
+      .config("spark.master", "local")
+      .getOrCreate()
+
+    // initialize a graph writer
+    val writer: GraphWriter = new GraphWriter()
+
+    // put movie graph data into writer
+    readAndPutDataIntoWriter(writer, spark)
+
+    // write in graphar format
+    val outputPath: String = args(0)
+    val vertexChunkSize: Long = args(1).toLong
+    val edgeChunkSize: Long = args(2).toLong
+    val fileType: String = args(3)
+
+    writer.write(outputPath, spark, "MovieGraph", vertexChunkSize, edgeChunkSize, fileType)
+}
 
 Additionally, when importing data from GraphAr to create/update instances in Neo4j, please refer to the following code:
 
 .. code:: scala
-  
-  // construct the GraphAr Spark reader
-  val spark = ...   // the Spark session
-  val prefix : String = ... // the prefix of the file path
-  val reader = new VertexReader(prefix, vertex_info, spark)
-
-  // reading chunks for all property groups
-  val vertex_df = reader.readAllVertexPropertyGroups(false)
-
-  // group vertices with the same Neo4j labels together
-  val labels_array = vertex_df.select("<labels>").distinct.collect.flatMap(_.toSeq)
-  val vertex_df_array = labels_array.map(labels => vertex_df.where(vertex_df("<labels>") === labels))
-
-  // write a group of vertices (with the same Neo4j labels) to Neo4j each time
-  vertex_df_array.foreach(df => {
-    val labels = df.first().getAs[Seq[String]]("<labels>")
-    var str = ""
-    labels.foreach(label => {str += ":" + label})
-    df.drop("<id>").drop("<labels>")
-      .write.format("org.neo4j.spark.DataSource")
-      .mode(SaveMode.Overwrite)
-      .option("labels", str)
-      .option("node.keys", "name")
-      .save()
-  })
+
+def main(args: Array[String]): Unit = {
+    // connect to the Neo4j instance
+    val spark = SparkSession.builder()
+      .appName("GraphAr to Neo4j for Movie Graph")
+      .config("neo4j.url", "bolt://localhost:7687")
+      .config("neo4j.authentication.type", "basic")
+      .config("neo4j.authentication.basic.username", sys.env.get("Neo4j_USR").get)
+      .config("neo4j.authentication.basic.password", sys.env.get("Neo4j_PWD").get)
+      .config("spark.master", "local")
+      .getOrCreate()
+
+    val graphInfoPath: String = args(0)
+    val graphInfo = GraphInfo.loadGraphInfo(graphInfoPath, spark)
+
+    val graphData = GraphReader.read(graphInfoPath, spark)
+    val vertexData = graphData._1
+    val edgeData = graphData._2
+
+    putVertexDataIntoNeo4j(graphInfo, vertexData, spark)
+    putEdgeDataIntoNeo4j(graphInfo, vertexData, edgeData, spark)
+}
 
 See `GraphAr2Neo4j.scala`_ for the complete example.
```

Comment:
Can you add some descriptions about how to write in different modes (i.e., how to modify the example).